error_response.go

```diff
@@ -6,24 +6,27 @@ import (
 	"strconv"
 )
 
+type Severity string
```

Comment:
This type doesn't seem to be used. Is it there for a reason?

Reply:
It’s cruft from a prior iteration. I’ll drop it and send a new version tomorrow.